aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,178 @@
 
-.. image:: pubIIS/_static/config-edit.png
-.. image:: pubIIS/_static/unlock.png
+Open the **Add Website** window.
 
-- Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
+	.. image:: pubIIS/_static/addwebsitecontextmenu.png
+	
+Configure the website.
 
- .. image:: pubIIS/_static/appPool.PNG
+	.. image:: pubIIS/_static/addwebsite.png
+	
+In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
 
+	.. image:: pubIIS/_static/basicsettingscontextmenu.png
+	
+Set the **.NET CLR version** to **No Managed Code**.
 
-Supported operating systems
------------------------------------
+	.. image:: pubIIS/_static/editapppool.png
 
-The following operations systems are supported:
+Create a Data Protection Registry Hive
+--------------------------------------
 
-- Windows 7 and newer
-- Windows 2008 R2 and newer
+Data Protection keys used by ASP.NET applications are stored in registry hives external to the applications. To persist the keys for a given application, you must create a registry hive for the application's application pool.
+
+For standalone IIS installations, you may use the `Data Protection Provision-AutoGenKeys.ps1 PowerShell script <https://github.com/aspnet/DataProtection/blob/dev/Provision-AutoGenKeys.ps1>`__ for each application pool used with an ASP.NET Core application. The keys will be persisted in the registry.
+
+In web farm scenarios, an application can be configured to use a UNC path to store its data protection key ring. By default, the data protection keys are not encrypted. You can deploy an x509 certificate to each machine to encrypt the key ring. See :ref:`Configuring Data Protection <data-protection-configuring>` for details.
+
+.. include:: ./dataProtectionWarning.txt
+
+Common Errors
+-------------
+
+The following is not a complete list of errors. Should you encounter an error not listed here, please leave a detailed error message in the DISQUS section below (click `Show comments` to open the DISQUS panel).
+
+To diagnose problems with IIS deployments, study browser output, examine the server's **Application** log through **Event Viewer**, and `enable logging <https://docs.asp.net/en/latest/fundamentals/logging.html>`__ in the application. The **ASP.NET Core Module** log will be found on the path provided in the `stdoutLogFile` attribute of the `\<aspNetCore\>` element in `web.config`. Any folders on the path provided in the attribute value must exist in the deployment. You must also set `stdoutLogEnabled="true"` to enable application logging. Applications that use the `publish-iis` tooling to create the `web.config` file will default the `stdoutLogEnabled` to `false`, so you must manually provide the file or modify the file in order to enable application logging.
```

Comment:
the name of the binary is `dotnet-publish-iis` since all dotnet commands have to start with the `dotnet-` prefix. However the tool is commonly invoked as `dotnet publihsh-iis` so I think `publish-iis` is fine.